ecosystem/typescript/sdk_v2/src/transactions/authenticator/transaction.ts

```diff
@@ -181,3 +182,35 @@ export class TransactionAuthenticatorFeePayer extends TransactionAuthenticator {
     return new TransactionAuthenticatorFeePayer(sender, secondary_signer_addresses, secondary_signers, fee_payer);
   }
 }
+
+export class TransactionAuthenticatorSecp256k1 extends TransactionAuthenticator {
+  public readonly public_key: Secp256k1PublicKey;
+
+  public readonly signature: Secp256k1Signature;
+
+  /**
+   * Transaction authenticator Secp256k1 for a single signer transaction
+   *
+   * @param public_key Client's public key.
+   * @param signature Secp256k1 signature of a raw transaction.
```

Comment:
```suggestion
   * @param signature Secp256k1 signature of a `RawTransaction`.
```

just to be clearer :p